s3/replication/common/src/s3replicationcommon/jobs.py

```diff
@@ -60,6 +67,105 @@ def get_keys(self):
     def reset(self):
         """Clear all jobs."""
         self._jobs.clear()
+        self._jobs_queued.clear()
+        self._jobs_inprogress.clear()
+        self._jobs_paused.clear()
+        self._jobs_completed.clear()
+
+    def get_queued(self, count):
+        """Get list of queued jobs."""
+        # Creates array reference, use cautiously w.r.t performance.
+        queued_list = []
+        for replication_id in self._jobs_queued.keys():
```

Comment:
just a corner case. : 
what if someone passes zero as the count? in this case it will fail.

Reply:
valid case, Fixed.